manifests/exports_data.pp

```diff
@@ -0,0 +1,20 @@
+define nfs::exports_data (
+  $config_title = "$title",
+  $message = 'This File is managed by Puppet - NFS Module',
```

Comment:
The header should be static and non-configurable. We use the first two lines from https://github.com/ghoneycutt/puppet-module-nfs/blob/master/files/exports#L1